samples/CosmosDbIntegtation/WriteAroundSamples/StreamSample.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WriteAroundSamples
+{
+    internal class StreamSample
+    {
+        // Redis database and stream stored in local.settings.json
+        private static readonly IDatabase redisDB = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable("redisConnectionString")).GetDatabase();
+        public const string stream = "streamTest";
+
+        // CosmosDB connection string, database name and container name stored in local.settings.json
+        public const string cosmosConnectionString = "cosmosConnectionString";
+        public const string cosmosDatabase = "%database-id%";
+        public const string cosmosContainer = "%container-id%";
+
+        /// <summary>
+        /// Write Around: Write from Cosmos DB to Redis whenever a change occurs in one of the CosmosDB documents
+        /// </summary>
+        /// <param name="input"> List of changed documents in CosmosDB </param>
+        /// <param name="logger"> ILogger used to write key information </param>
+        [FunctionName("CosmosToRedis")]
+        public static void Run(
+            [CosmosDBTrigger(

```

Comment:
Maybe we can rename this to `WriteAroundForStream` and the others to `WriteBehindForStream` so that when we merge all functions for Lists and PubSubs we do not have confusing names.